examples-site/react/example-client.tsx

```diff
@@ -0,0 +1,42 @@
+import { ElementType, StrictMode } from 'react'
+import { hydrateRoot } from 'react-dom/client'
+
+async function getComponent(): Promise<ElementType> {
+  const reExp = /^\/react\/(?<component>[\w|-]+)\/(?<exampleName>[\w|-]+)\/?$/
```

Comment:
Minor: I think the name `regex` for the variable would be more conventional and consistent:

```suggestion
  const regex = /^\/react\/(?<component>[\w|-]+)\/(?<exampleName>[\w|-]+)\/?$/
```